manim/_config/logger_utils.py

```diff
@@ -80,6 +82,9 @@ def make_logger(
     theme = parse_theme(parser)
     console = Console(theme=theme)
 
+    # With rich 9.5.0+ we could pass stderr=True instead
+    error_console = Console(theme=theme, file=sys.stderr)
```

Comment:
if you need, you can bump the minimum version to `9.5.0`

Reply:
Seeing as we can get the same functionality super easily with what we have now (and the thought of having to mess with poetry lol) I don't think it's necessary to bump the version, just thought I'd leave a comment for the future